pub_login.sh

```diff
@@ -23,7 +23,7 @@ if [ -z "${PUB_DEV_PUBLISH_EXPIRATION}" ]; then
   exit 1
 fi
 
-# Create credentials.json file.
+# Creat credentials.json file.
```

Comment:
typo error

Reply:
Should be fixed I think.

Reply:
@campfire5 ⬆️

Reply:
file was removed forever, there was no need of this for following reason, creating credentials.json file happens inside the publish.yml